Mathlib/RingTheory/Unramified/Basic.lean

```diff
@@ -214,4 +225,51 @@ theorem FormallyUnramified.localization_map [FormallyUnramified R S] :
 
 end Localization
 
+end FormallyUnramified
+
+section
+
+variable (R : Type u) [CommSemiring R]
+variable (A : Type u) [Semiring A] [Algebra R A]
+
+/-- An `R`-algebra `A` is unramified if it is formally unramified and of finite type. -/
+class Unramified : Prop where
+  formallyUnramified : FormallyUnramified R A := by infer_instance
```

Comment:
Is there a reason why this has an default argument (and not the other two)?

Reply:
no, thanks for spotting this